### PR TITLE
Joomla - Lazy Loading String Feedback

### PR DESCRIPTION
**Pull request by exterkamp**
This string was highlighted as being unclear by translators. This is a rewording suggestion.

### CODE REVIEW

File: packs/joomla.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const UIStrings = {
...
-  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. Starting with Joomla 4.0 the core is shipping an dedicated lazyloading plugin please enable the "Content - Lazy Loading Images" plugin. Also consider using [an AMP plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=amp).',
+  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. Starting with Joomla 4.0, the core is shipping a dedicated lazyloading plugin ("Content - Lazy Loading Images") which you should enable. Also consider using [an AMP plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=amp).',
```
Comment (paulirish) [inline packs/joomla.js:18]: cc @jeckodevelopment we're trying to fix the wording to make this more clear



from my understanding Joomla 4 is in beta but the final isn't shipped yet, right?

also... where can i read about these dedicated plugins that are bundled and shipped with joomla core?  looks like email-cloak and recaptcha are similar?

https://github.com/joomla/joomla-cms/pull/28838 is where this plugin was added.

cc @zero-24  to confirm: is this plugin shipped by default with joomla 4? also, it is disabled by default?


if a user were to enable it.. can you show a screenshot of where they'd find this?  i'm personally seeing the word "extension" more than the word "plugin".. so i want to be sure this is actually helping people.

Comment (paulirish) [inline packs/joomla.js:18]: here's my tweak.

```suggestion
  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. In Joomla 4.0, you can enable the "Content - Lazy Loading Images" core extension.',
```

**Review by paulirish — COMMENTED** [review]
thanks for raising this, shane.

**Comment by connorjclark** [comment]
I've edited the same string in https://github.com/GoogleChrome/lighthouse-stack-packs/pull/53/

File: packs/joomla.js (hunk 1 above)
Comment (zero-24) [reply packs/joomla.js:18]: Hi, thanks for picking this up. Right now there is that plugin in the core beta. But i'm not sure whether that will be the final solution for the core. We might should better say something like that joomla 4 will be shipped with lazyloading supported in the core.

Comment (zero-24) [reply packs/joomla.js:18]:
```suggestion
  'offscreen-images': 'Install a [lazy-load Joomla plugin](https://extensions.joomla.org/instant-search/?jed_live%5Bquery%5D=lazy%20loading) that provides the ability to defer any offscreen images, or switch to a template that provides that functionality. Starting with Joomla 4.0, all new images will automatically be get the loading attribute..',
```

@connorjclark @paulirish Here is another suggestion based on [this PR here](https://github.com/joomla/joomla-cms/pull/30748) for 4.x that drops that mention core plugin in favor of an deeper implementation that does no longer require an plugin to be setup.

Comment (zero-24) [reply packs/joomla.js:18]: I have just added the PR here: https://github.com/joomla/joomla-cms/pull/30748 for the Joomla Core which is going to drop the dedicated plugin in favor of an more deeper implementation that does not longer require an plugin.

I have also added an suggestion down for a better wording. Feel free to ping me on any questions.

**Comment by paulirish** [comment]
I've merged in everyone's awesome text changes and this reflects the latest and best from everyone.

thanks @zero-24  in particular for making this functionality automagically on by default!  definitely makes it easier for us to tell folks to use it. :)

**Pull request merged**